python/nav/ipdevpoll/jobs.py

```diff
@@ -81,7 +81,8 @@ class JobHandler(object):
 
     def __init__(self, name, netbox, plugins=None, interval=None):
         self.name = name
-        self.netbox = netbox
+        self.netbox_id = netbox
```

Comment:
You may be able to fix it by populating the `netbox` attribute before initializing any other classes in the constructor, instead of deferring it to the `run` method.